charts/dave/values.yaml

```diff
@@ -81,6 +81,7 @@ backend:
       ELASTICSEARCH_REINDEX: "true"
       ELASTICSEARCH_SCHEME: http
       # Dave
+      DB_SCHEMA: "dave"
```

Comment:
redundant  https://github.com/it-at-m/helm-charts/pull/108/files#diff-177fa935390658f0b17aceed9e4dde3138e50dc718e4986aaa25fc674cf94970R39